packages/grape-web-client/src/components/browser-notification/BrowserNotification.js

```diff
@@ -181,46 +184,6 @@ const normalizeNotificationData = ({ dispatcher, props, conf }) => {
   return undefined
 }
 
-const updateNotification = (props, nextProps) => {
-  const {
-    conf,
-    incomingCall,
-    notification,
-    browserNotification: { dispatcher },
-  } = nextProps
-
-  const { type } = normalizeNotificationData({
-    dispatcher,
-    props,
-    conf,
-  })
-
-  if (type === 'calls') {
-    const { show } = incomingCall
-    if (!show && show !== props.incomingCall.show && notification.close) {
-      notification.close()
-    }
-  }
-}
-
-const renderNotification = props => {
-  const { browserNotification, conf } = props
-  const { dispatcher } = browserNotification
-
-  const { properties, callbacks, params } = normalizeNotificationData({
-    dispatcher,
-    props,
-    conf,
-  })
-
-  const notification = createNotification({
-    properties,
-    params,
-    callbacks,
-  })
-  return notification
-}
-
```

Comment:
renderNotification is not needed anymore?

Reply:
No, I refactored the code a bit. It using createNotification underhood